app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -129,6 +125,7 @@ def analytics_properties
         multi_factor_auth_method: 'backup_codes',
         in_multi_mfa_selection_flow: in_multi_mfa_selection_flow?,
         enabled_mfa_methods_count: mfa_context.enabled_mfa_methods_count,
+        referer: ParseControllerFromReferer.new(request.referer).call,
```

Comment:
The existing `analytics_properties` seem to have been relevant for creation actions. I don't think all of them should apply to the "visit" event.

We could either have some logic in this method to differentiate, or just separate methods (e.g. `visit_analytics_properties` or `analytics_properties_for_visit`, etc).

Reply:
Understood. I have moved that to its own more appropriately named method.